### PR TITLE
fixed toggle and data loading bug

### PR DESCRIPTION
**Pull request by GraceGardner**
What (if any) features are you implementing?
- added toggle to our dropdown buttons to make them go away when a selection is made.
- made the homepage update upon round finish and redirect.

What (if anything) did you refactor?
- leagueMember Provider
- Score card
- dropdown menu

Were there any issues that arose?
N/A

Is there anything that you need from your teammate?
Review

Any other comments, questions, or concerns?
N/A

### CODE REVIEW

File: src/Contexts/LeagueMembersProvider.js [hunk 1]
```diff
@@ -6,19 +6,22 @@ export const LeagueMembers = createContext()
 
 export const LeagueMembersProvider = ({ children }) => {
   const { errorMessage, setErrorMessage } = useContext(Errors)
-
   const [ leagueMembers, setLeagueMembers ] = useState([])
 
-  useEffect(() => {
+  const getLeagueMembers = () => {
```
Comment (oacortez) [inline src/Contexts/LeagueMembersProvider.js:11]: Thank you for fixing this bug, so all proper the players stats update properly

File: src/Components/Navbar.js [hunk 2]
```diff
@@ -30,9 +30,9 @@ const Navbar = () => {
           <FontAwesomeIcon icon={ faBars }/>
         </button>
       </nav>
-      {clicked && <DropdownMenu />}
+      {clicked && <DropdownMenu toggleClick={toggleClick}/>}
```
Comment (philalewis) [inline src/Components/Navbar.js:33]: Excell

File: src/Components/DropdownMenu.js [hunk 3]
```diff
@@ -2,15 +2,15 @@ import React from 'react'
...
 import '../Styles/DropdownMenu.scss'
 
-const DropdownMenu = () => {
+const DropdownMenu = ({toggleClick}) => {
 
   return(
     <div className='dropdown-container'>
       <Link to="/manage/">
-        <button className='manage-link-btn'>MANAGE LEAGUE</button>
+        <button onClick={toggleClick} className='manage-link-btn'>MANAGE LEAGUE</button>
```
Comment (philalewis) [inline src/Components/DropdownMenu.js:10]: Excellent. I'm glad we got this to toggle. Much better ui.

File: src/Components/DropdownMenu.js [hunk 4]
```diff
@@ -2,15 +2,15 @@ import React from 'react'
 import { Link } from 'react-router-dom'
 import '../Styles/DropdownMenu.scss'
 
-const DropdownMenu = () => {
+const DropdownMenu = ({toggleClick}) => {
```
Comment (oacortez) [inline src/Components/DropdownMenu.js:5]: Thank you for fixing this toggle bug we had!

**Review by oacortez — APPROVED** [review]

File: src/Contexts/LeagueMembersProvider.js (hunk 1 above)
Comment (philalewis) [inline src/Contexts/LeagueMembersProvider.js:11]: Good that it is reusable now.

**Review by philalewis — APPROVED** [review]

**Pull request merged**